src/Instance.h

```diff
@@ -30,6 +30,10 @@ void WriteOurPid();
 /// Throws if PID file maintenance is disabled.
 pid_t Other();
 
+/// Returns MD5 hash of the PID file name to make a uniq service name
+/// to use it in the shared memory segments naming
```

Comment:
We do not need to disclose the implementation details like being MD5. Just stating its a hash is fine.
```suggestion
/// \returns hash of the PID file name to make a unique service name for this Squid instance
```